Euclidean_Space.R

```diff
@@ -0,0 +1,183 @@
+#Euclidean Space R

```

Comment:
Please apply Google R style convention for all file names, variable names and function names:
https://google.github.io/styleguide/Rguide.xml